src/app/groups/create-group/create-group.component.html

```diff
@@ -0,0 +1,131 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="groupForm">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="groupForm.controls.name.hasError('required')">
+              Group Name is <strong>required</strong>
+            </mat-error>
+            <mat-error *ngIf="groupForm.controls.name.hasError('pattern')">
+              Group Name <strong>cannot</strong> begin with a special character or number
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Office</mat-label>
+            <mat-select required formControlName="officeId">
+              <mat-option *ngFor="let office of officeData" [value]="office.id">
+              {{ office.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="groupForm.controls.officeId.hasError('required')">
+              Office is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Staff</mat-label>
+            <mat-select formControlName="staffId">
+              <mat-option *ngFor="let staff of staffData" [value]="staff.id">
+              {{ staff.displayName }}
```

Comment:
You can deactivate the staff field if staffData.length === 0. That will provide a better user experience.